### PR TITLE
Update homescreen listviews

### PR DESCRIPTION
**Pull request by MeganFantes**
Updated the 3 listviews in the homescreen to populate like we want them to. Wait to merge this until we can test if they populate with transaction data.

Also, all of the localization stuff shows up in this PR :/ Sorry about that.
Important files for this PR have javadoc comment at the top:
/**
  IMPORTANT FILE FOR THIS PR
 */
i.e. Look at Home Fragment, OwnedItemsAdapter, and RentingPendingItemsAdapter

### CODE REVIEW

File: app/src/main/java/com/teamcaffeine/hotswap/navigation/ProfileFragment.java [hunk 1]
```diff
@@ -112,6 +114,15 @@ public void hideProgressDialog() {
         }
     }
 
+    // Safely get the payment the user has selected
+    public String getSelectedPayment() {
```
Comment (alexrosenfeld10) [inline app/src/main/java/com/teamcaffeine/hotswap/navigation/ProfileFragment.java]: I'm pretty sure the getSelected function of the list already returns null if there's nothing selected. No harm done by having it here though.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -94,7 +93,7 @@
 
     <!--Invite friends strings-->
     <string name="invite_message">Hey I\'ve been using HotSwap to rent items that I need but don\'t want to buy! Check it out!</string>
-    <string name="post_to_FB_url">https://developers.facebook.com</string>
+    <string name="post_to_FB_url" translatable="false">https://hotswap.com</string>
```
Comment (alexrosenfeld10) [inline app/src/main/res/values/strings.xml]: uhh. does this  even exist

Comment (MeganFantes) [reply app/src/main/res/values/strings.xml]: lol no, URL is there to make the FB post look a little more legit but it doesn't do anything

File: app/app.iml [hunk 3]
```diff
@@ -230,5 +228,6 @@
     <orderEntry type="library" name="com.android.support:support-core-utils-27.0.2" level="project" />
     <orderEntry type="library" name="com.google.firebase:firebase-analytics-impl-license-11.6.0" level="project" />
     <orderEntry type="library" name="com.google.firebase:firebase-analytics-license-11.6.0" level="project" />
+    <orderEntry type="library" name="android-android-26" level="project" />
```
Comment (tkixi) [inline app/app.iml]: why is this being added now or did it exist before

File: app/src/main/java/com/teamcaffeine/hotswap/navigation/ProfileFragment.java [hunk 4]
```diff
@@ -131,43 +142,39 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
-        // populate the listview with the user's addresses
```
Comment (tkixi) [inline app/src/main/java/com/teamcaffeine/hotswap/navigation/ProfileFragment.java]: why no more comments on the new one

File: app/app.iml (hunk 3 above)
Comment (alexrosenfeld10) [reply app/app.iml]: not sure.. android does it's own weird thing to app.iml, no clue what the deal is

File: app/src/main/java/com/teamcaffeine/hotswap/navigation/ProfileFragment.java [hunk 5]
```diff
@@ -177,40 +184,22 @@ public void onItemClick(AdapterView<?> parent, View view, final int position, lo
...
                                 DatabaseReference ref = users.child(firebaseUser.getUid());
-                                // add a single value event listener to the user reference to
-                                // delete the single item from their account
                                 ref.addListenerForSingleValueEvent(new ValueEventListener() {
-                                    // get a datasnapshot of the current user to access its data
                                     @Override
                                     public void onDataChange(DataSnapshot dataSnapshot) {
-                                        // create a user object from the datasnapshot
                                         User user = dataSnapshot.getValue(User.class);
-
```
Comment (tkixi) [inline app/src/main/java/com/teamcaffeine/hotswap/navigation/ProfileFragment.java]: where did all the pretty comments go

File: app/app.iml (hunk 3 above)
Comment (MeganFantes) [reply app/app.iml]: Last night I was debugging, and an IDE warning popped up saying I didn't have android 26 downloaded. At that point I couldn't find the bug, so I thought it was an android studio problem so I downloaded level 26

File: app/src/main/java/com/teamcaffeine/hotswap/navigation/ProfileFragment.java (hunk 5 above)
Comment (MeganFantes) [reply app/src/main/java/com/teamcaffeine/hotswap/navigation/ProfileFragment.java]: they got in the way of my flow, yo (also we had changed some things that made a large portion of the comments irrelevant/redundant)

File: app/src/main/java/com/teamcaffeine/hotswap/navigation/homeAdapters/OwnedItemsAdapter.java [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+        return items.get(position);
+    }
+
+    public void putItem(Item item) {
+        if (item != null) {
+            this.items.add(item);
+        }
+    }
+
+    public void clear() {
```
Comment (williamieong) [inline app/src/main/java/com/teamcaffeine/hotswap/navigation/homeAdapters/OwnedItemsAdapter.java]: im sad this isn't called .nuke()

Comment (williamieong) [reply app/src/main/java/com/teamcaffeine/hotswap/navigation/homeAdapters/OwnedItemsAdapter.java]: LOL this is just a comment, ignore it as a review

File: app/src/main/java/com/teamcaffeine/hotswap/navigation/homeAdapters/RentingPendingItemsAdapter.java [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+            LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+            row = inflater.inflate(R.layout.listview_row, parent, false);
+        }
+
+        else
+        {
+            row = convertView;
+        }
+
+        TextView itemName = (TextView) row.findViewById(R.id.itemTitle);
```
Comment (williamieong) [inline app/src/main/java/com/teamcaffeine/hotswap/navigation/homeAdapters/RentingPendingItemsAdapter.java]: We can add it later, but I'm thinking for renting/pending transactions we should also be able to see who its from (i.e im renting an iron from Megan)

File: app/src/main/java/com/teamcaffeine/hotswap/navigation/homeAdapters/OwnedItemsAdapter.java (hunk 6 above)
Comment (MeganFantes) [reply app/src/main/java/com/teamcaffeine/hotswap/navigation/homeAdapters/OwnedItemsAdapter.java]: I can't steal your and Tony "thing" ;)

File: app/src/main/java/com/teamcaffeine/hotswap/navigation/homeAdapters/RentingPendingItemsAdapter.java (hunk 7 above)
Comment (MeganFantes) [reply app/src/main/java/com/teamcaffeine/hotswap/navigation/homeAdapters/RentingPendingItemsAdapter.java]: Oh good idea. Yeah let's add it later, right now let's just make sure the functionality works.

**Pull request merged**